### ISSUE TITLE
[Feature request]: Add command line control

### ISSUE THREAD

**Issue by brunetton** [vzhd1701/gridplayer#61]
### Suggestion

Hi

I really love this soft, and I'm using it often, to compare video compression options. But this is long to manually click to let my grid looks like I want:
- set aspects ratios
- set grid orientation
- set sync between videos (thanks for this !)

I'd love to be able to make a super little script that calls Gridplayer with arguments; something like:

    GridPlayer --aspect none -- video1.mp4 video2.mp4

I know that I can set default behavior, but sometimes I need different behaviors.

Would you be open to a PR adding a command line interface, using [Docpie](https://github.com/TylerTemp/docpie) (Docopt alternative) ?

**Comment by parkan** [comment]
I'm not actually even able to load files at all from the command line using the AppImage, passing in a file path gives a "no playable media or urls" error in the UI

the fact that the error comes up makes me think that it does parse the paths but perhaps something with the way AppImages are jailed makes them not resolve?

can you confirm whether you can in fact pass videos on the command line and which version you're using?

**Comment by justpeanuts** [comment]
Yes please.. my use case that made me look for gridplayer was to play many video files simultaneously for a presentation. During a presentation you cannot make the audience wait for you to startup the gui and set things up.. a commandline interface is needed so that i can press a hotkey which runs a script and the window is setup just like a want it to be.
